VirtusLab-Open-Source/strapi-plugin-navigation
# feat: navigation item wrapper type restored

## Ticket

N/A

## Summary

What does this PR do/solve? 

`Wrapper` navigation item type restored

## Test Plan

- start server
- display navigation tree
- add new item
- wrapper item should be available
- save
- read navigation through API
- `wrapper` type items should be rendered
